spec/views/report/_form_expression_buttons.html.haml_spec.rb

```diff
@@ -0,0 +1,27 @@
+describe 'report/_form_filter.html.haml' do
+  before :each do
+    report = FactoryGirl.create(:miq_report)
+    @expkey = :record_filter
+    exp_table = [["Example AFTER \"value1\"", 1]]
+    expression = {"after" => {"field" => "field_example", "value" => "value1"}, :token => 1}
+    @edit = {:rpt_id => report.id,
+             @expkey => {:exp_table => exp_table, :expression => expression}}
+  end
```

Comment:
For setting up the context, you should use [`let`](http://www.betterspecs.org/#let)